assets/queries/dockerfile/npm_install_without_pinned_version/query.rego

```diff
@@ -0,0 +1,89 @@
+package Cx
+
+CxPolicy [ result ] {
+  runCmd := input.document[i].command[name][_]
+  isRunCmd(runCmd) 
+  
+  value := runCmd.Value
+  count(value) == 1 #command is in a single string
+
+  cmd := value[0]
+
+  searchIndex := indexof(cmd,"npm install")
```

Comment:
Should check for 'npm i' and 'npm add' as well

Reply:
fixed